src/steamship/client/steamship.py

```diff
@@ -183,19 +186,53 @@ def use_plugin(
             fetch_if_exists=fetch_if_exists,
         )
 
+    def use_skill(
+        self,
+        skill: Skill,
+        provider: Optional[Vendor] = None,
+        instance_handle: Optional[str] = None,
+        fetch_if_exists: Optional[bool] = True,
+    ) -> PluginInstance:
+
+        if skill not in SKILL_TO_PROVIDER:
+            raise SteamshipError(
+                f"Unsupported skill provided. "
+                f"Use one of our supported skills: {','.join(SKILL_TO_PROVIDER)}"
+            )
+
+        if provider and provider not in SKILL_TO_PROVIDER[skill]:
+            raise SteamshipError(
+                f"The provider {provider} has no support for the skill {skill}."
+                f"Use one of the providers that support your skill: "
+                f"{','.join(SKILL_TO_PROVIDER[skill])}"
+            )
+
+        plugin_setup = (
+            SKILL_TO_PROVIDER[skill][provider]
+            if provider
+            else list(SKILL_TO_PROVIDER[skill].values())[0]
```

Comment:
Is the order of .values() guaranteed stable?  Do we want to have an explicit 'default' vendor in the map?

Reply:
.values() should in theory be stable. The keys are sorted by order of insertion since Python 3.7 ✅